### PR TITLE
Update filetype mapping and references to non-recommended icon sources (for Fabric7)

### PR DESCRIPTION
**Pull request by bigbadcapers**
#### Pull request checklist

- [x] Folding in PR from Fabric 6: https://github.com/microsoft/fluentui/pull/12363
- [x] Added more extension support to cover Teams mobile app pre-existing extensions
- [x] Add PowerBI App and new raw image format extension support to the filetype-icon map
- [x] Replaced references to filetype icons in documentation pages that use deprecated icons from brand-icons or other non-recommended folders to only use the official ones.
- [x] Removing any lingering references to deprecated iconsets under *-fluent folders
- [x] Include a change request file using `$ yarn change` 
- [x] yarn update-snapshots

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12422)

### CODE REVIEW

**Comment by bigbadcapers** [comment]
Hi @aneeshack4 -- let me know if you want to have a quick chat about this PR and its goals so I can help unblock anything required to be able to approve it without much slowdown. Thanks!

**Comment by bigbadcapers** [comment]
@ecraig12345 did I do that right ? :)

**Comment by bigbadcapers** [comment]
@ecraig12345 @aneeshack4 please let me know what work is left here, thank you!

File: change/@uifabric-example-data-2020-04-06-10-28-34-caperez-filemap-mar2020-fab7.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "comment": "removing more references to wrong icon-ppt.png icon in wrong folder",
```
Comment (ecraig12345) [inline change/@uifabric-example-data-2020-04-06-10-28-34-caperez-filemap-mar2020-fab7.json:3]: Can you check the messages for each change file and make sure they properly correspond to the changes in that particular package?

**Pull request merged**